xbmc/guilib/GUIIncludes.h

```diff
@@ -53,6 +53,10 @@ class CGUIIncludes
 
 private:
   void ResolveIncludesForNode(TiXmlElement *node, std::map<INFO::InfoPtr, bool>* xmlIncludeConditions = NULL);
+  std::map<std::string, std::string> GetParameters(const TiXmlElement *includeCall, const TiXmlElement *includeDef) const;
+  void ResolveParametersForNode(TiXmlElement *node, const std::map<std::string, std::string> &params) const;
+  enum RESOLVE_PARAMS_RESULT { NO_PARAMS_FOUND, PARAMS_RESOLVED, SINGLE_UNDEFINED_PARAM_RESOLVED };
```

Comment:
Please put the enum above all method definitions with every enum value on a separate line. It's very easy to miss here.